symbols/arm9.yml

```diff
@@ -1091,6 +1133,22 @@ arm9:
         r0: log level
         r1: format
         ...: variadic
+    - name: InitDebugStripped4
+      address:
+        NA: 0x200C24C
+      description: "Does nothing, only called in the debug initialisation function."
+    - name: InitDebugStripped3
+      address:
+        NA: 0x200C250
+      description: "Does nothing, only called in the debug initialisation function."
+    - name: InitDebugStripped2
+      address:
+        NA: 0x200C254
+      description: "Does nothing, only called in the debug initialisation function."
+    - name: InitDebugStripped1
+      address:
+        NA: 0x200C258
+      description: "Does nothing, only called in the debug initialisation function."
```

Comment:
Maybe these descriptions should be changed to "initiali**z**ation" for consistency?

Reply:
I wasn't sure if we were enforcing a style of British/American spelling. If there is a preferred style then I can make a follow-up PR to fix this.

Reply:
I don't think there's a spelling style, but I did a quick search and most of the repo uses initialize. Might want to wait and see what @UsernameFodder has to say, but I'm guessing it's a good idea to keep it consistent, specially since `DEBUG_IS_INITIALIZED` was just introduced in this same PR.

Reply:
I've made a PR for this: https://github.com/UsernameFodder/pmdsky-debug/pull/190

Reply:
I don't really enforce it strictly as a matter of respect for the European researchers, but if given the choice I do generally prefer American spelling for consistency